config/initializers/feature_toggles.rb

```diff
@@ -9,3 +9,6 @@
     beta_testers.include?(user.email)
   end
 end
+OpenFoodNetwork::FeatureToggle.enable(:unit_price) do |user|
+  beta_testers.include?(user.email)
+end
```

Comment:
@sauloperez 
I just had a look to https://github.com/openfoodfoundation/ofn-install/pull/704 and if I understood correctly, each user of FR instance is now considered as a `beta_tester` which is quite embarrassing for the unit_price feature. Actually, as I coded, I'm waiting for a list of emails.
I'm wondering if we should consider to use a `env` variable per feature? Such as `UNIT_PRICE_BETA_TESTERS` and have a unique management system for each case?

Reply:
but notice that due to that the condition `beta_testers.include?(user.email)` will always evaluate to false. Indeed, it's something we need to solve differently but it won't reveal the feature.

Reply:
- I've created an issue to solve the _list of emails_: #6870
 - For the second point, I guess I should have a look at `ENV['RAILS_ENV']` or something like that? 

Moving to `In Dev`.... ;-)

Reply:
Modification is done: https://github.com/openfoodfoundation/openfoodnetwork/pull/6851/commits/406c18051146169b6de567bc8b8cf84a5a1ec0e5#diff-0f07cf1c313099baff76848102decb21d55a569b13e0aedb05f6b855f684cf35R14

I hope this is what was expected.

Moving back to `Code Review` waiting for @sauloperez approval.